apps/ejabberd/src/ct_progress_hook.erl

```diff
@@ -0,0 +1,67 @@
+%%% @doc Common Test Example Common Test Hook module.
+-module(ct_progress_hook).
```

Comment:
Can we have this module in `apps/ejabberd/test` instead of `src`? It better suits there.